stack.yaml

```diff
@@ -0,0 +1,67 @@
+# This file was automatically generated by 'stack init'
```

Comment:
FWIW, I tend to strip out all the comments in files like this.

Reply:
Done!